Overview.src.html

```diff
@@ -1851,10 +1886,54 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   for HTTP/2, and equivalent information used to prioritize dispatch and processing of
   HTTP/1 fetches.
 
+
+ <li><p>If <var>request</var> is a <span>navigation request</span>, a user agent should <span title=concept-header-list-append>append</span> the following pairs to <var>request</var>'s
+   <span title=concept-request-header-list>header list</span>:
```

Comment:
I wonder if we should do the for each here too, that might make it read a little better.

A user agent should, for each header name /x/ in the first column of the following of the following table, if /x/ is not in request's header list, append /x/ / a value given in the same row on the second column, to request's header list.